### ISSUE TITLE
Add term distal radioulnar joint dislocation

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10409]
**Preferred term label:**
Distal radioulnar joint dislocation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Distal radioulnar joint dislocation is more commonly displaced dorsally (i.e. the distal ulna is dorsally dislocated with respect to the distal radius), but can be volarly displaced PMID: 23162140

**Parent term (use hpo.jax.org/app)**
Radioulnar dislocation HP:0006439

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Galeazzi and Essex-Lopresti fracture

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000889
- label: Distal radioulnar joint dislocation
- parents: [Radioulnar dislocation (HP:0006439)](https://hpo.jax.org/app/browse/term/HP:0006439)
- pmid: PMID:23162140
- def: Distal radioulnar joint dislocation is more commonly displaced dorsally (i.e. the distal ulna is dorsally dislocated with respect to the distal radius), but can be volarly displaced.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10409
- synonyms:
